src/main/scala/nodes/nlp/POSTagger.scala

```diff
@@ -0,0 +1,16 @@
+package nodes.nlp
+
+import epic.sequences.{CRF, Segmentation, TaggedSequence}
+import epic.trees.AnnotatedLabel
+import workflow.Transformer
+import breeze.linalg.Counter2
+import org.apache.spark.rdd.RDD
+
+case class POSTagger(@transient tagger: CRF[AnnotatedLabel, String])
```

Comment:
Hmm.. won't this be skipped when the `POSTagger` is serialized as part of a spark job?